mirdata/beatles.py

```diff
@@ -193,6 +294,8 @@ def _load_sections(sections_path):
 
 
 def _fix_newpoint(beat_positions):
+    """(placeholder)
```

Comment:
Fills in missing beat position labels by inferring the beat position from neighboring beats.